db/structure.sql

```diff
@@ -1,26 +1,14 @@
 SET statement_timeout = 0;
 SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
```

Comment:
aku verziu pg mas? nejako vela to pomenilo.

Reply:
4